client/src/pages/login.tsx

```diff
@@ -1,13 +1,14 @@
-import { useLogin } from "@refinedev/core";
+import { useLogin } from "@CarRentalServices/core";
```

Comment:
Please do not change the import statements, they are not causing any error. Revert the old statements!